cardano-api/src/Cardano/Api.hs

```diff
@@ -22,6 +22,7 @@ module Cardano.Api
   , byronPubKeyAddress
   , byronGenKeyPair
   , shelleyGenKeyPair
+  , randomShelleyKeyPair
```

Comment:
Not sure if that should be exported, especially not with that name. I would hope all key generations is random :D.

Can I suggest `genericShelleyKeyPair` and I still do not think it should be exported.

Reply:
Will do.